packages/validator/src/commands/convertCmd.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
```

Comment:
Under the hood they are the same thing, there was a need for a conversion tool, rather to build it from ground up it was more efficient to extend already existing validator. It is pretending to be 2 different things on the surface not to cause confusion, it just makes more sense to separate validation and conversion as it might be quite counter-intuitive to run validation command when you want to convert your OAS to OIS. Keep in mind users that have no idea how it works and what validator templates are might be using this tool.
Validator should not return `output`, this was bug and it was fixed in a70ef65.

Reply:
> so that our own templates will be used instead of some provided custom ones, right?


`from`/`to` in convertor is just to help users, so they don't have to look for common conversions, but ofc they can make their own conversion templates and run it as mentioned in the readme: 
`npm run convert --template="templates/3.0.0/OAS2OIS.json" --specs="exampleSpecs/OAS.specs.json"`